### DISCUSSION TITLE
New Mapmaker Here

### DISCUSSION THREAD

**Discussion by JackHubert** [triplea-game/triplea#10863] [New Contributor]
New map maker here. When i click the tutorial button shown below it says file not found in github. Where do i go from here?

![readme](https://user-images.githubusercontent.com/109051709/178176917-de4e2b6d-5bf3-4a1a-906b-9d9257b532fd.JPG)

**Comment by panther2** [comment]
@JackHubert 

When I click the "Start Tutorial" button I see:
![git1](https://user-images.githubusercontent.com/20812775/178206458-ab81db73-99fe-4130-b432-daabd51380e2.png)
___
After clicking that link I see:
![git2](https://user-images.githubusercontent.com/20812775/178206687-5318fa43-b6b2-4059-a84c-866e676febe9.png)
___
And klicking "tutorial" brings me to the two documents:
![git3](https://user-images.githubusercontent.com/20812775/178206923-4e7cb014-7409-4bb6-8408-0337e66f8334.png)
___
Klick on the filename to see the content.

**Comment by TheDog-GH** [comment]
@JackHubert
Just in case you needed the urls
https://github.com/triplea-game/triplea/wiki/How-to-create-custom-maps
https://github.com/triplea-game/triplea/tree/master/docs/map-making

**Reply by JackHubert** [reply]
Thank you. I'll give this a go
